### PR TITLE
POLIO-637: Filter out test campaigns in the excel export

### PR DESCRIPTION
**Pull request by hakifran**
The XLSX calendar export includes test campaigns too.

## Self proof reading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [ ] My migrations file are included
- [x] Are there enough tests
## Changes
- Filter rounds base on the campaign is_test boolean field by including only rounds with campaign is_test False

## How to test
- Add a test campaign
- Export calendar as an XLSX file

## Print screen / video
Peek 2022-10-12 11-41

### CODE REVIEW

File: plugins/polio/api.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from collections import defaultdict
 from functools import lru_cache
 from logging import getLogger
+from typing_extensions import reveal_type
```
Comment (niconoe) [inline plugins/polio/api.py]: Is that a forgotten import?

File: plugins/polio/api.py [hunk 2]
```diff
@@ -246,31 +246,50 @@ def get_year(current_date):
...
         if campaign_type == "preventive":
             rounds = rounds.filter(campaign__is_preventive=True)
-        if campaign_type == "test":
-            rounds = rounds.filter(campaign__is_test=True)
         if campaign_type == "regular":
             rounds = rounds.filter(campaign__is_preventive=False).filter(campaign__is_test=False)
         if search:
             rounds = rounds.filter(Q(campaign__obr_name__icontains=search) | Q(campaign__epid__icontains=search))
-
+        print(rounds)
```
Comment (niconoe) [inline plugins/polio/api.py]: Forgotten?

File: plugins/polio/api.py [hunk 3]
```diff
@@ -246,31 +246,50 @@ def get_year(current_date):
             return today.year
 
     @staticmethod
```
Comment (niconoe) [inline plugins/polio/api.py]: I don't think this method should be static (we can discuss in 1-1 if you want!)

File: plugins/polio/api.py [hunk 4]
```diff
@@ -246,31 +246,50 @@ def get_year(current_date):
...
-    def get_calendar_data(self, year, params):
+    def get_calendar_data(self: Any, year: int, params: Any) -> Any:
+        """
+        Returns filtered rounds from database
+
+            parameters:
+                self (CampaignViewSet): a self CampaignViewSet
```
Comment (niconoe) [inline plugins/polio/api.py]: The self parameter does need need explanation IMHO (given that we are in a non-static method)

File: plugins/polio/export_utils.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
-from openpyxl import Workbook
-from openpyxl.styles import Alignment, Font, PatternFill
-from openpyxl.styles.borders import Border, Side
+from openpyxl import Workbook  # type: ignore
+from openpyxl.styles import Alignment, Font, PatternFill  # type: ignore
+from openpyxl.styles.borders import Border, Side  # type: ignore
 import datetime as dt
```
Comment (niconoe) [inline plugins/polio/export_utils.py]: Maybe we can move that to the mypy config file as discussed earlier?

**Review by niconoe — CHANGES_REQUESTED** [review]
Hi Franc, thanks a lot fr the great work! Looks very good in general, I let a few comments.!

File: plugins/polio/api.py (hunk 3 above)
Comment (hakifran) [reply plugins/polio/api.py]: Fix in : 
- https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R226
- https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R226

File: plugins/polio/export_utils.py (hunk 5 above)
Comment (hakifran) [reply plugins/polio/export_utils.py]: Fixed in:
https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R226

File: plugins/polio/api.py (hunk 4 above)
Comment (hakifran) [reply plugins/polio/api.py]: Fixed in : https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R285

File: plugins/polio/api.py (hunk 2 above)
Comment (hakifran) [reply plugins/polio/api.py]: fixed in : https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R277

File: plugins/polio/api.py (hunk 1 above)
Comment (hakifran) [reply plugins/polio/api.py]: Fixed in : https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R10

**Comment by niconoe** [comment]
Thanks @hakifran, great work! If I'm not mistaken, those recent changes are currently in the other PR only. Can you also fix them on this branch/PR, so we can merge them separately as suggested by Christophe in the squad? Thanks!

**Comment by hakifran** [comment]
> Thanks @hakifran, great work! If I'm not mistaken, those recent changes are currently in the other PR only. Can you also fix them on this branch/PR, so we can merge them separately as suggested by Christophe in the squad? Thanks!

Thank you @niconoe , Done in this PR

**Review by niconoe — APPROVED** [review]

**Pull request merged**